frontend/package.json

```diff
@@ -148,13 +148,14 @@
     "@patternfly/quickstarts": "5.1.0",
     "@patternfly/react-catalog-view-extension": "5.0.0",
     "@patternfly/react-charts": "7.1.1",
+    "@patternfly/react-component-groups": "5.1.0-prerelease.5",
     "@patternfly/react-console": "5.0.0",
-    "@patternfly/react-core": "5.1.1",
-    "@patternfly/react-icons": "5.1.1",
+    "@patternfly/react-core": "^5.1.1",
```

Comment:
@vojtechszocs figured out the `^` is required on these @patternfly dependencies so that multiple versions of them do not get pulled in while adding `@patternfly/react-component-groups`.  Many thanks, @vojtechszocs!